docs/mevblocker/users-and-integrators/users/sending-transactions-via-rpc.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 3
 
 # Sending Transactions via RPC
 
-MEV Blocker supports sending transactions via the standard `eth_sendRawTransaction` method.
+MEV Blocker supports sending transactions via multiple endpoints (as discussed [here](https://docs.cow.fi/mevblocker/concepts/endpoints)) using the standard `eth_sendRawTransaction` method.
```

Comment:
```suggestion
MEV Blocker supports sending transactions via multiple endpoints (as discussed [here](/mevblocker/concepts/endpoints)) using the standard `eth_sendRawTransaction` method.
```

Please don't use absolute links as this will break things if we need to move the documentation site in the future :pray: